### ISSUE TITLE
Website Accessibility-WCAG 4.1.2: Nested interactive controls are not announced by screen readers (li[aria-controls="example-1-scss"])

### ISSUE THREAD

**Issue by A11Ybot-OP** [sass/sass#3231]
#### Issue
`Nested interactive controls are not announced by screen readers` ([nested-interactive](https://dequeuniversity.com/rules/axe/4.3/nested-interactive?application=msftAI))
#### Target application
[Sass: Sass Basics](https://sass-lang.com/guide)
#### Element path
li[aria-controls="example-1-scss"]
#### Snippet
`<li class="ui-tabs-tab ui-tabs-active ui-corner-top ui-state-default ui-tab ui-state-active" role="tab" tabindex="0" aria-controls="example-1-scss" aria-labelledby="ui-id-1" aria-selected="true" aria-expanded="true">`
#### How to fix
    Fix any of the following:
      Element has focusable descendants
#### Environment
Microsoft Edge version 96.0.1054.62
